state/watcher/txnwatcher.go

```diff
@@ -327,11 +326,6 @@ func (w *TxnWatcher) loop() error {
 			logCollection = w.getTxnLogCollection()
 		}
 		added, err := w.sync(logCollection)
-		if wrench.IsActive("txnwatcher", "sync-error") {
```

Comment:
What we could do is augment the check with a cheap logger.IsTraceEnabled() call.
So this allows the wrench to still be turned on when needed to test, but short circuits the filesystem access.

Reply:
Agreed. I'll update this patch accordingly.